lib/rules/no-capital-letters-in-routes.js

```diff
@@ -5,10 +5,12 @@ const ember = require('../utils/ember');
 // Routing - No capital letters in routes
 //------------------------------------------------------------------------------
 
+const PATTERN = /^[A-Z]{1}/;
```

Comment:
is the `{1}` necessary here?

Reply:
I don't know (as in, the code works now, but it may also work if `{1}` is omitted since I haven't tried).

Are you asking a question, or implying that it isn't necessary?

Reply:
I'm assuming that it isn't necessary since `{1}` should be the default already

Reply:
Ya, agreed. Lets remove the `{1}` here...

Reply:
I think it's better to make the range-length explicit. There are always ways to [shorten code](https://codegolf.stackexchange.com/), but I prefer to optimize on readability (less cognitive overhead for other developers).

Reply:
so if you want to test against "hello" you write the regex `/h{1}e{1}l{2}o{1}/`?